joomla/joomla-cms
# The help screen system isn't a CMS extension anymore

#### Summary of Changes

The primary help screen system is no longer a CMS extension and the replacement application does not use the `option=com_help` query fragment so we can stop setting it as part of the help screen system's URLs for the default configuration.
#### Testing Instructions

For purposes of validating this patch, code review is all that is necessary for the SQL files.

For other test cases, first you'll want to re-save your global configuration.  This will cause the `helpurl` variable to be saved with the new value (without the `option=com_help` query fragment).  After that, clicking any help button in the backend should cause a new window to pop up and notice that in that URL the `option=com_help` query fragment isn't there but you still get the correct output.

To check the behavior on the "Help > Joomla! Help" menu, you'll need your browser's dev console open to inspect network requests.  Clicking on any of the links on the left menu should get you that help page and validate the request does not have the `option=com_help` fragment.

Note: Though this will "fix" #11381 by removing the need for the ampersand in the request URL, it's really not a proper fix as it's just bypassing whatever change happened to cause the ampersand to stop being decoded.
